scripts/core/components/ListItem.tsx

```diff
@@ -5,6 +5,58 @@ import classNames from 'classnames';
 import {IListItemProps, IPropsListItemColumn, IPropsListItemRow} from 'superdesk-api';
 
 export class ListItem extends React.Component<IListItemProps> {
+    private clickTimeout: number | null = null;
+    private readonly clickDelay = 250;
+
+    constructor(props: IListItemProps) {
+        super(props);
+
+        this.handleClick = this.handleClick.bind(this);
+        this.handleDoubleClick = this.handleDoubleClick.bind(this);
+    }
```

Comment:
you can get rid of the whole constructor if you adjust the declaration of your handlers like this below (you won't need the `bind`

```tsx
handleClick = (event: React.MouseEvent<HTMLDivElement>) => {
```